Fixes #87

harper-wavra opened the pull request:
changed to
=(0.42/SQRT(H19))(H17H16/SQRT(H18/100))^0.8
from
=(0.42/SQRT(H19))(H17H16/SQRT((MAX(0.0001,H18)/100))^0.8)

harper-wavra committed:
#87 - edit sheetflow equation

amedenblik reviewed (changes requested):
The `traveltimetc` endpoint is giving this 500 error: 
```
{
  "detail": "float division by zero"
}
```

Upon further inspection of the spreadsheet, it turns out that the value of the Bermuda grass Overland Slope is not actually 0.00 but 0.000001: 
![image](https://user-images.githubusercontent.com/40237491/211924637-2b1e69c9-db50-478c-a65f-1d844f040695.png)

That is what is preventing the spreadsheet from having a similar "divide by 0" error.

My suggestion is to change the equation to:

~`travel_time_sheet_flow += (0.42 / math.sqrt(P2_24_2)) * (mannings_n*corrected_length/math.sqrt((max(0.000001,surface["Overland Slope"])/100.0))**0.8) # minutes`~

Need to look into this more.

harper-wavra committed:
#87 - added exception, updated example request

harper-wavra committed:
#87 - Update CHANGELOG.md

harper-wavra commented:
- added Exception if user enters a value for Overland Slope that is less than or equal to 0
- changed traveltimetc example request to match new spreadsheet

amedenblik commented on CHANGELOG.md:
```diff
@@ -9,12 +9,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added 
 
--  
+-  Exception if user enters a value for Overland Slope that is less than or eqaul to 0
```
Typo "eqaul"

amedenblik replied:
```suggestion
-  Exception if user enters a value for Overland Slope that is less than or equal to 0
```

harper-wavra committed:
Update CHANGELOG.md

Co-authored-by: Andrea S. Medenblik <40237491+amedenblik@users.noreply.github.com>
